tuf/api/metadata.py

```diff
@@ -182,6 +182,20 @@ def from_bytes(
 
         return deserializer.deserialize(data)
 
+    def to_bytes(
+        self, serializer: Optional[MetadataSerializer] = None
+    ) -> bytes:
+        """Return the bytes representation of self."""
```

Comment:
Please improve this
* arguments
* raises
* I think even the return value is not yet well defined: it's not just "self as bytes", it's the serialized TUF file format as bytes)

Reply:
I updated it.